### DISCUSSION TITLE
[功能建议]调整直播页面在竖屏时的弹幕位置

### DISCUSSION THREAD

**Discussion by chenxi-Eumenides** [the1812/Bilibili-Evolved#4179] [功能建议 / Ideas]
### 功能描述
当页面为竖屏（或者较长）时，将弹幕放至直播画面的下方，如同视频页面的评论区一般。

### 功能扩展
1.可选网页全屏时更改页面排版或者默认时就更改。
2.可选开启该功能时，直播画面处于页面的中间/靠上/靠下。
3.可选弹幕处于直播画面的上方/下方。

###  缘由
使用竖屏看直播时，如果无论使用何种方式，直播画面和弹幕栏总是处于左右并排的样式，这样就挤占了原本就不宽的竖屏宽度，导致弹幕和直播画面都无法看清。

### 脚本版本
v2.7.1-49-g261f1b61a

### 附加截图
大概类似与这样？

![QQ截图20230504222200](https://user-images.githubusercontent.com/76398980/236236546-a4aa44fe-7665-4520-b626-d0a1ec71e1eb.jpg)

### 致遇到了相同问题准备回复的人
**太长不看**：
1、遇到了相同的问题：**请用左下角的 reaction 点赞**
2、实时关注该 discussion 的后续进展：**请用 discussion 页上的 Subscribe 功能**（在 PC 端按钮通常位于页面右侧）。
<details>
  <summary>完整版</summary>
    由于每个 discussion 甚至整个项目都是可以订阅通知的, 对于已经有人汇报过的 Bug, 如果您也遇到了并且没有其他信息要补充, 可以直接在左下点个赞表示 +1, 不要再去新增评论。想关注后续的可以用 Subscribe，Subscribe 之后该 discussion 如果有变动都可以收到通知，如果不想被回复打扰可以使用更加进阶的 Customize 功能，勾选 "Closed" & "Reopened"。Closed 通常代表该 Issue 已被解决，极少数情况下的 Reopened 代表该 Issue 的问题重新出现。如果您新增了无意义评论，因为 discussion 这个 bug tracker 与<a href="https://www.zhihu.com/question/23757640">邮件列表</a>类似，订阅某个 discussion 的用户会在每次这个 discussion 有新的动态时收到一封新的提醒邮件。也就是说，在 discussion 中的一个不能提供新的信息的评论，相当于给所有订阅这个 discussion 的用户发了一封垃圾邮件（spam）。尤其项目的维护者也是订阅者中的一员，太多垃圾邮件会对维护者带来很多不必要的困扰。发送垃圾邮件在开源社区通常是不那么受欢迎的行为。
</details>

**Comment by the1812** [comment]
我有个没有放出来过的功能, 不知道能不能解决你的问题, 就是网页全屏的时候将弹幕区半透明悬浮在直播画面上
![image](https://user-images.githubusercontent.com/26504152/236844070-e7eff250-28af-4a22-97e1-41d5170b8281.png)
![image](https://user-images.githubusercontent.com/26504152/236844231-51117dee-ffa6-49af-80fe-b015500d3a00.png)
鼠标经过的时候会恢复不透明
![image](https://user-images.githubusercontent.com/26504152/236845170-4e57b272-7dde-4bfd-8512-0078490cc2f1.png)

**Reply by chenxi-Eumenides** [reply]
这看起来挺帅的，虽然不能替代我说的方案，但貌似不管横屏竖屏都挺有用。
看起来已经很完善了，如果能拖动弹幕栏或选择位置，那这个功能就基本完美了。
想要/流口水
大佬牛逼/流口水
